setup.py

```diff
@@ -316,7 +316,7 @@ def run(self):
           'numpy>=1.16',
           'pandas==0.25.3',
           'pyarrow==0.15.1',
-          'numba>=0.48'
+          'numba >=0.49.0,<0.50.0'
```

Comment:
Why not 0.49.1?

Reply:
For not be binded to 0.49.1 when 0.49.2 will be released.

Reply:
So then probably it would be better `> 0.49.0` ?

Reply:
B/c 0.50.0 could break backward compatibility.

Reply:
`< 0.49.0, <0.50.0`

Reply:
But 0.49.0 is also workable.

Reply:
Numba 0.49.0 has significant regressions, it is better not to use it and start from 0.49.1.